### ISSUE TITLE
QT Voice apps dissapear

### ISSUE THREAD

**Issue by timchez** [thomas-barthelemy/QQ-LoL-Localizer#8]
So after installing applying localizer all QT panel disappears from the lower bar near friend list.
![285738841356548943](https://cloud.githubusercontent.com/assets/20891077/17464322/5072aee0-5d0e-11e6-9dc2-49fbf9392fe4.jpg)
The picture is WITHOUT applying localizer on the client yet. But after applying it all disappears. So how to fix this issue?

**Comment by thomas-barthelemy** [comment]
My understanding is that QQ made his own AIR Client on top, which is made by/for China only and (at least at the time) isn't translatable. What is done is a simple modification of configurations so that the Intl. air client is used instead. Sadly, indeed, it does not have the special features added by QQ like the voice. This will won't be solvable until they decide to open their client for translation but then this software won't be needed anymore =)
I'll leave that task open as a reminder to check again if anything changed since last time I checked.

tl;dr: Expected, probably nothing possible.
